functional-tests/constants.js

```diff
@@ -13,3 +13,14 @@ exports.SELECTOR_BOX_PREVIEW_DOC = '.bp-doc';
 exports.SELECTOR_BOX_PREVIEW_MP3 = '.bp-media-mp3';
 exports.SELECTOR_BOX_PREVIEW_DASH = '.bp-media-dash';
 exports.SELECTOR_BOX_PREVIEW_MP4 = '.bp-media-mp4';
+exports.SELECTOR_BOX_PREVIEW_ZOOM_OUT = '.bp-doc-zoom-out-icon';
+exports.SELECTOR_BOX_PREVIEW_ZOOM_IN = '.bp-doc-zoom-in-icon';
```

Comment:
if some of these are buttons could we label them as such? Might make reading the test easier. Also could probably shorten BOX_PREVIEW to bp.

Reply:
@jeremypress I will add the button, but perhaps the removal of BOX_PREVIEW should be another PR as it will be quite large